### PR TITLE
Updated question and university migration and association

### PR DESCRIPTION
**Pull request by uakubue**
Here, I've actually added model for question, successfully implemented association between university and questions.

### CODE REVIEW

File: app/models/question.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Question < ApplicationRecord
+    belongs_to :university
```
Comment (obi-a) [inline app/models/question.rb:2]: This should be indented with two spaces

File: app/models/university.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class University < ApplicationRecord
+    has_many :questions
```
Comment (obi-a) [inline app/models/university.rb:2]: Should indent two spaces, that's the ruby convention.

**Review by obi-a — COMMENTED** [review]
Looks good! Just added small comment for indenting with 2 spaces.

File: app/models/question.rb (hunk 1 above)
Comment (uakubue) [reply app/models/question.rb:2]: Thank you Bro

File: app/models/university.rb (hunk 2 above)
Comment (uakubue) [reply app/models/university.rb:2]: Okay, I've captured that.

**Review by obi-a — APPROVED** [review]

**Pull request merged**